README.md

```diff
@@ -21,6 +21,18 @@ can be still be installed with the old name by specifying
 
 TODO
 
+## API Keys
+
+The Delphi API will soon require a registered API key for unthrottled queries.
```

Comment:
suggestion:
- "registered API key" -> "(free) registered API key" / "(free) API key" --- does "registered" mean associated with someone's background or their name?
- "for unthrottled queries" -> ~"to issue very frequent or complex queries"?  "for unthrottled queries" could read as some special advanced type of query